services/backend/api/api.py

```diff
@@ -1,8 +1,40 @@
 from fastapi import FastAPI
+from fastapi.middleware.cors import CORSMiddleware
+from .models import Condition, Forecast
+from typing import List, Optional
+from datetime import datetime, timedelta
+import random
 
 app = FastAPI()
 
+origins = ["http://localhost", "http://localhost:3000"]
+
+app.add_middleware(
```

Comment:
The frontend is technically located at a different origin so we have to enable CORS so the requests will be allowed. [More here](https://fastapi.tiangolo.com/tutorial/cors/).